frame/assets/src/lib.rs

```diff
@@ -184,6 +194,8 @@ pub struct AssetDetails<
 	zombies: u32,
 	/// The total number of accounts.
 	accounts: u32,
+	/// Whether the asset is frozen for permissionless transfers.
+	is_frozen: bool,
```

Comment:
might be useful to have two error types then: `AssetFrozen` and `AccountFrozen`?

Reply:
meeh, i see no reason to add this tbh. The error returning frozen is functionally the same in both cases